File: Integreatedcode/src/app/model/applicant.ts
import { Cibil } from './cibil';
import { LoanDetails } from './loan-details';
import { PermanentAddress } from './permanent-address';
import { CurrentAddress } from './current-address';
import { Customer } from './customer';
import { PropertyDetails } from './property-details';
import { Document } from './document';
import { CustomerBankDetails } from './customer-bank-details';
export class Applicant {
   applicationId: string;
   education: string;
    nationality: string;
    profession: string;   
    customer: Customer;
    caddr: CurrentAddress;
    paddr: PermanentAddress;	
    loanDetails: LoanDetails;	
    propertydetails: PropertyDetails;   
    bankdetails: CustomerBankDetails;
     verificationResult: string;
    
   
}

File: Integreatedcode/HomeLoanBackEndNew/src/main/java/com/wcs/homeloan/app/model/Applicant.java
package com.wcs.homeloan.app.model;

import javax.persistence.CascadeType;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.OneToOne;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Applicant {
	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	private String applicationId;
	private String education;
	private String nationality;
	private String profession;
	@OneToOne(cascade = CascadeType.ALL)
	private Customer customer;
	@OneToOne(cascade = CascadeType.ALL)
	private CurrentAddress caddr;
	@OneToOne(cascade = CascadeType.ALL)
	private PermanentAddress paddr;
	@OneToOne(cascade = CascadeType.ALL)
	private LoanDetails loanDetails;
	@OneToOne(cascade = CascadeType.ALL)
	private PropertyDetails propertyDetails;
	@OneToOne(cascade = CascadeType.ALL)	
	private CustomerBankDetails bankDetails;	
	private String verificationResult="Pending";
	

	

}
